lib/cli/src/detect.test.ts

```diff
@@ -159,6 +159,16 @@ const MOCK_FRAMEWORK_FILES = [
       },
     },
   },
+  {
+    name: ProjectType.WEB_COMPONENTS,
+    files: {
+      'package.json': {
+        dependencies: {
+          'lit-html': '1.0.0',
```

Comment:
should this be an exact dep?

Reply:
It's just for the test fixture ;) any version is ok

Reply:
ok